src/Middleware/RequestThrottling/RequestThrottling.slnf

```diff
@@ -0,0 +1,25 @@
+{
+  "solution": {
```

Comment:
What is this? @natemcmaster

Reply:
+ @halter73

Reply:
It's a saved VS solution filter that only loads the parts of the sln listed here. The Middleware.sln is too large to load and work with, but creating an sln for each middleware is also overkill. Creating a filter for each middleware looks like a promising compromise.
https://docs.microsoft.com/en-us/visualstudio/ide/filtered-solutions?view=vs-2019

Reply:
+1. I've actually been thinking maybe we should only have one .sln which includes everything, and then just use a bunch of .slnf files instead....but that's not at the top of my priority list right now.